include/pq_flash_index.h

```diff
@@ -78,8 +78,6 @@ namespace diskann {
 
     DISKANN_DLLEXPORT void load_cache_list(std::vector<uint32_t> &node_list);
 
-    DISKANN_DLLEXPORT _u64 return_nd();
```

Comment:
Why is return_nd() removed?

Reply:
It's just a declaration without definition, AFAIR Intellisense issued a warning to that effect.